code/modules/tgui/states/notcontained.dm

```diff
@@ -4,20 +4,20 @@
  * Checks that the user is not inside src_object, and then makes the default checks.
  */
 
-GLOBAL_DATUM_INIT(notcontained_state, /datum/ui_state/notcontained_state, new)
+// GLOBAL_DATUM_INIT(notcontained_state, /datum/ui_state/notcontained_state, new)
```

Comment:
for all of these TGUI states, can you just use a multiline comment inside of commenting out each individual line?